src/cli/enhancedChat.ts

```
@@ -0,0 +1,537 @@
+			case 'toollist':
+				this.showAvailableTools();
+				break;
+			case 'export':
+				await this.exportConversation();
+				break;
+			case 'exit':
+			case 'quit':
+				console.log('👋 Goodbye!');
+				process.exit(0);
+			default:
+				await this.sendMessage(input);
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)